src/main/scala/com/fulcrumgenomics/bam/api/SamRecord.scala

```diff
@@ -227,7 +227,7 @@ trait SamRecord {
   @inline final def remove(name: String): Unit                   = setAttribute(name, null)
 
   // transient attributes
-  @inline final def transientAttrs: TransientAttrs = new TransientAttrs(this)
+  @inline final val transientAttrs: TransientAttrs = new TransientAttrs(this)
```

Comment:
This is unrelated, but IntelliJ kept getting confused and thinking uses of `rec.transientAttrs[String]("foo")` were errors, and this is likely better anyway.